pvoutput/pvoutput.py

```diff
@@ -1103,16 +1104,28 @@ def _process_api_response(self, response: requests.Response) -> str:
         # If we get to here then the content is valid :)
         return content
 
-    def wait_for_rate_limit_reset(self):
+    def wait_for_rate_limit_reset(self, do_sleeping: bool = True) -> int:
+        """
+        Wait for reset limit
+
+        Args:
+            do_sleeping: bool to do the sleeping, or not.
+
+        Returns: The number of seconds needed to sleep
+        """
         utc_now = pd.Timestamp.utcnow()
         timedelta_to_wait = self.rate_limit_reset_time - utc_now
         timedelta_to_wait += timedelta(minutes=3)  # Just for safety
         secs_to_wait = timedelta_to_wait.total_seconds()
         retry_time_utc = utc_now + timedelta_to_wait
+        retry_time_local = retry_time_utc.tz_convert(tz=datetime.now(tzlocal()).tzname())
```

Comment:
Two quick thoughts:

First, please consider adding a short comment to explain why we need the reset time to be in local time

Second, I wonder if `tzlocal()` will be guaranteed to return the "correct" timezone, even if this code is running in the cloud (possibly on a server that's in a wildly different timezone)?